migrations/data.sql

```diff
@@ -0,0 +1,205 @@
+
+DO $$
+--
+-- A list of uuids generated using `select gen_random_uuid()`.
+-- 
+-- Note: We could also generate UUID values automatically using
+--       `DEFAULT` constraint and then assign UUID to a variable with the use
+--       of `RETURNING id into <variable>`, but this is more convenient
+--       as UUIDs will stay same after each execution of this SQL scriopt.
+--
+DECLARE
+  company0_id UUID := 'b5188eda-528d-48d4-8cee-498e0971f9f5';
+  company1_id UUID := '134d5286-5f55-4637-9b98-223a5820a464';
+  company2_id UUID := '71fa27d6-6f00-4ad0-8902-778e298aaed2';
+
+  user0_id UUID := '35341253-da20-40b6-96d8-ce069b1ba5d4';
+  user1_id UUID := '0465041f-fe64-461f-9f71-71e3b97ca85f';
+  user2_id UUID := 'ac9bf689-a713-4b66-a3d0-41faaf0f8d0c';
+  user3_id UUID := '51a01dbf-dcd5-43a0-809c-94ed8e61d420';
+
+  event0_id UUID := 'b71fd7ce-c891-410a-9bb4-70fc5c7748f8';
+  event1_id UUID :='3f152d12-0bbd-429a-a9c5-28967d6370cc';
```

Comment:
Actually, I just noticed there is no Event1

Reply:
Yea, thanks for comment, some of these variables are just not utilized yet ;).